sdks/python/apache_beam/runners/interactive/utils.py

```diff
@@ -0,0 +1,55 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Utilities to be used in  Interactive Beam.
+"""
+
+from __future__ import absolute_import
+
+import pandas as pd
+
+from apache_beam.utils.windowed_value import WindowedValue
+
+
+def elements_to_df(elements, include_window_info=False):
+  """Parses the given elements into a Dataframe.
+
+  If the elements are a list of `WindowedValue`s, then it will break out the
+  elements into their own DataFrame and return it. If include_window_info is
+  True, then it will concatenate the windowing information onto the elements
+  DataFrame.
+  """
+
+  rows = []
+  windowed_values = []
```

Comment:
windowing_info? (windowed_values sounds like it has the values themselves).

Reply:
Changed to "windowing_info"